test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -178,11 +173,13 @@ test
     deviceChallengeFalllbackPage.clickOktaVerifyButton();
     const deviceChallengePollPageObject = new DeviceChallengePollPageObject(t);
     await t.expect(deviceChallengePollPageObject.getBeaconClass()).contains(BEACON_CLASS);
-    await t.expect(deviceChallengePollPageObject.getHeader()).eql('Extra verification needed');
-    await t.expect(deviceChallengePollPageObject.getContent()).eql('' +
-      'Your organization needs extra verification to make sure it\'s you signing in.\n' +
-      'To continue, confirm your screen lock in Okta Verify.\n' +
-      'Confirm screen lock');
+    await t.expect(deviceChallengePollPageObject.getHeader()).eql('Sign in using Okta Verify on this device');
+    await t.expect(deviceChallengePollPageObject.getSpinner().getStyleProperty('display')).eql('block');
+    await t.expect(deviceChallengePollPageObject.getPrimiaryButtonText()).eql('Reopen Okta Verify');
+    await t.expect(deviceChallengePollPageObject.getFooterLink().exists).eql(false);
+    await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().exists).eql(false);
+    await t.expect(deviceChallengePollPageObject.getFooterSwitchAuthenticatorLink().innerText).eql('Verify with something else');
```

Comment:
I think we also need to click on the switch authenticator link and verify it can go back to select authenticator page. https://github.com/okta/okta-signin-widget/pull/1554/files#diff-698534168a87ff567c25b350509b7261a3660769a3c5fd36b10adc3873731f39R306